examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -41,7 +41,7 @@
 
 #include "ble_config.h"
 
-#if SL_ICD_ENABLED && SIWX_917
```

Comment:
@lpbeliveau-silabs we will need this define since 917 NCP and 917 SoC uses the same files

Reply:
All the SIWX_917 or any M4 define since it is not needed for 917 NCP

Reply:
Undid un-necessary removal